deploy/kubernetes/filebeat/filebeat-configmap.yaml

```diff
@@ -14,11 +14,11 @@ data:
         # Mounted `filebeat-prospectors` configmap:
         path: ${path.config}/prospectors.d/*.yml
         # Reload prospectors configs as they change:
-        reload.enabled: false
+        reload.enabled: true
```

Comment:
@exekias Is this is a breaking change from your perspective? If yes, will remove it from the PR.

Reply:
I would say it is. +1 to keep it to `false` by default, rest of the PR LGTM

Reply:
Sorry, I realize now this is the configmap :man_facepalming: , I'm ok with having this enabled by default